README.md

```diff
@@ -1 +1,179 @@
-# Capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>My README File</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Capstone Project] <a name="Foursquare Annual Convention 2023 Namibia"></a>
+
+**[Foursquare Annual Convention]** is an online website for an Annual Convetion conference that happens every year at Foursquare Gospel church in Namibia. This same event also happens at Foursquare Gospel Churches in other countries world wide. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML, CSS AND JAVASCRIPT
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Mobile version]**
+- **[Desktop version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://wilhelmk109.github.io/my-portfolio-website/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project presentation video <a name="project-presentation"></a>
+
+- [Video Link](https://www.loom.com/share/0702dde6b8814e24b9b60b3b84852f69)
```

Comment:
- [ ] I kindly suggest that you recreate your video because you are missing the part where you showcase a codpiece that you are proud of. Doing so would make your presentation more exciting and professional.